ec2/ec2-create-instance.sh

```diff
@@ -60,6 +68,25 @@ if [ ! -z "$BRANCH" ]; then
    echo "building branch $BRANCH"
 fi
 
+# The AMI ID may change in the future and the way to look it up is with the following command, which takes a long time to run:
+# aws ec2 describe-images  --owners 'aws-marketplace' --filters 'Name=product-code,Values=aw0evgkw8e5c1q413zgy5pjce' --query 'sort_by(Images, &CreationDate)[-1].[ImageId]' --output 'text'
+# To use an AMI, one must subscribe to it via the AWS GUI.
+# AMI IDs are specific to the region.
```

Comment:
I see from the comment above that this "describe-images" command is slow but in a future pull request should we consider adding a flag to execute it? It's helpful to have it here as a comment for now, at least.